### PR TITLE
#23 create zustand store and examples.

### PR DESCRIPTION
**Pull request by ShravanSunder**
- create zustand store and services folder
- created example slices.  used example in tempContract
- Additionally, moved web3 folder into new services folder

### CODE REVIEW

File: packages/frontend/services/store/slices/anotherExampleTempState.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { TTempState as TAnotherTempState } from "./tempExampleState";
+
+export type TAnotherTempState2 = {
+  tempStuff: string;
+};
```
Comment (carletex) [inline packages/frontend/services/store/slices/anotherExampleTempState.ts:5]: I guess we don't need that for this example?

Seems we are importing the Type from the other example

File: packages/frontend/services/store/store.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  tempState: defaultTempState(),
+  setTempState: (newTempState: TTempState): void =>
+    set((state): TAppStore => {
+      state.tempState = newTempState;
+      return state;
+    }),
+});
+
+const createAnotherTempStateSlice: TAppSliceCreator<AnotherTempStateSlice> = set => ({
+  tempState: defaultTempState(),
```
Comment (carletex) [inline packages/frontend/services/store/store.ts:21]: Should it be "defaultTempState()" ??

File: packages/frontend/services/store/storeTypes.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { StateCreator } from "zustand";
+import { TempStateSlice } from "~~/services/store/slices/tempExampleState";
+import { AnotherTempStateSlice } from "~~/services/store/slices/anotherExampleTempState";
+
+/**
+ * The App store
+ * All your slices of state should be joined here
+ */
+
```
Comment (carletex) [inline packages/frontend/services/store/storeTypes.ts:9]: Remove this line? :D

**Review by carletex — CHANGES_REQUESTED** [review]
This amazing @ShravanSunder great work!

I think reading the in-code comments that you wrote (+ when we have real examples) would be enough for people to understand how they should add new slices. Or do you think need to add a little README under `services/store`?

File: packages/frontend/services/store/slices/anotherExampleTempState.ts (hunk 1 above)
Comment (ShravanSunder) [reply packages/frontend/services/store/slices/anotherExampleTempState.ts:5]: i fixed it, was a mistake

**Comment by ShravanSunder** [comment]
> This amazing @ShravanSunder great work!
> 
> I think reading the in-code comments that you wrote (+ when we have real examples) would be enough for people to understand how they should add new slices. Or do you think need to add a little README under `services/store`?

I added a readme!

**Review by carletex — APPROVED** [review]
lgtm!

Thanks for the Readme :)

**Pull request merged**